Documentation/gettingstarted/servicemesh/installation.rst

```diff
@@ -9,33 +9,67 @@ Prerequisites
 Installation
 ############
 
-Cilium Ingress Controller can be enabled with helm flag ``ingressController.enabled``
-set as true.
+.. tabs::
 
-.. parsed-literal::
+    .. group-tab:: Helm
 
-    $ helm upgrade cilium |CHART_RELEASE| \\
-        --namespace kube-system \\
-        --reuse-values \\
-        --set ingressController.enabled=true
+        Cilium Ingress Controller can be enabled with helm flag ``ingressController.enabled``
+        set as true. Please refer to :ref:`k8s_install_helm` for a fresh installation.
 
-If you only want to use envoy traffic management feature without Ingress support, you should only
-enable ``--enable-envoy-config`` flag.
+        .. parsed-literal::
 
-.. parsed-literal::
+            $ helm upgrade cilium |CHART_RELEASE| \\
+                --namespace kube-system \\
+                --reuse-values \\
+                --set ingressController.enabled=true
 
-    $ helm upgrade cilium |CHART_RELEASE| \\
-        --namespace kube-system \\
-        --reuse-values \\
-        --set-string extraConfig.enable-envoy-config=true
 
-Next you can check the status of the Cilium agent and operator:
+        If you only want to use envoy traffic management feature without Ingress support, you should only
+        enable ``--enable-envoy-config`` flag.
 
-.. code-block:: shell-session
+        .. parsed-literal::
 
-    $ cilium status
+            $ helm upgrade cilium |CHART_RELEASE| \\
+                --namespace kube-system \\
+                --reuse-values \\
+                --set-string extraConfig.enable-envoy-config=true
```

Comment:
Setting `ingressController.enabled=true` already sets `enable-envoy-config=true` accordingly to https://github.com/cilium/cilium/blob/7463d9f12ba643d707f95a587ea03107ae46f0cb/install/kubernetes/cilium/templates/cilium-configmap.yaml#L210-L212

So we can remove this paragraph.